dockerfiles/Dockerfile.shellcheck

```diff
@@ -0,0 +1,9 @@
+FROM debian:stretch-slim
```

Comment:
I noticed that as well. The problem with `debian:jessie` is the version of Shellcheck. (See: https://github.com/moby/moby/pull/33260#issuecomment-302330346).

I would have to recompile from source which involves `cabal-install`. I'm not sure it's worth the effort but I understand those warnings annoy me as well :)